consul/tests/common.py

```diff
@@ -23,31 +23,30 @@
 
 # Not all the metrics are exposed in this test environment.
 PROMETHEUS_METRICS = [
-    'consul.memberlist.gossip.count',
-    'consul.memberlist.gossip.sum',
     'consul.memberlist.msg.alive',
-    'consul.memberlist.probenode.count',
-    'consul.memberlist.probenode.sum',
-    'consul.memberlist.pushpullnode.count',
-    'consul.memberlist.pushpullnode.sum',
     'consul.memberlist.tcp.accept',
     'consul.memberlist.tcp.connect',
     'consul.memberlist.tcp.sent',
     'consul.memberlist.udp.received',
     'consul.memberlist.udp.sent',
-    'consul.raft.commitTime.count',
-    'consul.raft.commitTime.sum',
-    'consul.raft.leader.dispatchLog.count',
-    'consul.raft.leader.dispatchLog.sum',
     'consul.raft.state.candidate',
     'consul.raft.state.leader',
-    'consul.serf.coordinate.adjustment_ms.count',
-    'consul.serf.coordinate.adjustment_ms.sum',
     'consul.serf.events',
     'consul.serf.member.join',
     'consul.serf.member.update',
-    'consul.serf.msgs.received.count',
-    'consul.serf.msgs.received.sum',
-    'consul.serf.msgs.sent.count',
-    'consul.serf.msgs.sent.sum',
+    'consul.raft.leader.lastContact.count',
+]
+
+PROMETHEUS_HIST_METRICS = [
+    'consul.memberlist.gossip.',
+    'consul.memberlist.probenode.',
+    'consul.memberlist.probenode.',
+    'consul.memberlist.pushpullnode.',
+    'consul.raft.commitTime.',
+    'consul.raft.leader.dispatchLog.',
+    'consul.raft.replication.appendEntries.rpc.',
```

Comment:
What about `raft.replication.installSnapshot` and `raft.replication.appendEntries.logs`? Do we also want to `.assert_metric()` them as well?

Reply:
AFAIK, these metrics are only available in the follower instances and we are testing the leader, so I guess I should add an integration test for a follower

Reply:
Added a comment to explain that these 2 metrics are not exposed in our testing environment